lib/tasks/configure/get-prompts.js

```diff
@@ -54,6 +54,16 @@ module.exports = function getPrompts(config, argv, environment) {
             });
         }
 
+        if (!argv.dbport) {
```

Comment:
I don't think we need this option here - the database port is something that you generally don't have to configure for most Ghost-CLI setups, and as such adding a prompt for it only adds unnecessary steps that the user has to follow in order to setup a blog successfully. Just adding the dbport option in the file below this should be enough for this PR.